R/expr-expr.R

```diff
@@ -3904,7 +3904,9 @@ expr__fill_nan <- function(value) {
 #' )
 expr__fill_null <- function(value, strategy = NULL, limit = NULL) {
   wrap({
-    check_exclusive(value, strategy)
+    if (!missing(value) && !missing(strategy) && !is.null(value) && !is.null(strategy)) {
+      abort("Exactly one of `value` or `strategy` must be supplied.")
+    }
```

Comment:
This is necessary because when we call this `fill_null()` from the LazyFrame method, `value` is something and `strategy` is `NULL`, which fails this check.

It is annoying that `check_exclusive()` fails if both values are `NULL` or one is something and the other is `NULL`, how about we make a variant of this function that allows those cases?

Reply:
I think that makes sense.

Reply:
Done in [`d9c70ed`](https://github.com/eitsupi/neo-r-polars/pull/158/commits/d9c70edf58a563e3d6f153a7ba0439290ababa7e)